helion/_testing.py

```diff
@@ -51,6 +51,16 @@ def skipIfRocm(reason: str) -> Callable[[Callable], Callable]:
     return unittest.skipIf(torch.version.hip is not None, reason)  # pyright: ignore[reportAttributeAccessIssue]
 
 
+def isRocm() -> bool:
+    """Check if PyTorch is running on ROCm (AMD GPU)."""
+    return torch.version.hip is not None  # pyright: ignore[reportAttributeAccessIssue]
+
+
+def isCuda() -> bool:
+    """Check if PyTorch is running on CUDA (NVIDIA GPU)."""
+    return torch.version.hip is None  # pyright: ignore[reportAttributeAccessIssue]
+
```

Comment:
We shouldn't assume cuda without checking the device.type.  Use `is_rocm` for helper functions.